server/server.go

```diff
@@ -61,6 +61,25 @@ type Server struct {
 	defaultIDMappings *idtools.IDMappings
 
 	updateLock sync.RWMutex
+
+	// pullCache is used to avoid pulling the same image in parallel. Goroutines
+	// will block on the pullResult.
+	pullCache map[string]*pullOperation
```

Comment:
Isn’t this more of a `pullsInProgress` than a “cache”?

Reply:
Renamed to `pullOperationsInProgress` :+1: